### PR TITLE
Connect Notification to Call Notes Create

### PR DESCRIPTION
**Pull request by peterp**

### CODE REVIEW

**Comment by peterp** [comment]
@justinvdm let me know what you think of this?

**Comment by justinvdm** [comment]
@peterp exactly what I had in mind :) I think the call completed container isn't give the activity id at the moment, so I think we'll need to use [`react-redux's` `mergeProps`](https://github.com/reactjs/react-redux/blob/master/docs/api.md#connectmapstatetoprops-mapdispatchtoprops-mergeprops-options) or do something in the container to get the activity id.

**Comment by justinvdm** [comment]
screen shot 2016-10-12 at 10 09 17 pm

**Comment by justinvdm** [comment]
Ready for review.

File: src/reducers/callNote.js [hunk 1]
```diff
@@ -22,7 +22,18 @@ const callNote = (state = {}, action) => {
...
+const reduce = combineReducers({
   navigation,
   callNote,
 });
+
+
+export default (state = {}, action) => {
+  switch (action.type) {
+    case constants.OPEN_CALL_NOTE_CREATE:
+      return reduce({}, action);
```
Comment (peterp) [inline src/reducers/callNote.js]: Awesome!

**Comment by peterp** [comment]
👍

File: src/views/CallNoteStepper/index.js [hunk 2]
```diff
@@ -1,40 +1,42 @@
...
+  call,
+  steps,
   navigationState,
-  callId,
-  activityId,
 }) => {
-  if (activityId) {
+  if (isUndefined(call.activity)) {
     return (
       <Stepper navigationState={navigationState}>
```
Comment (justinvdm) [inline src/views/CallNoteStepper/index.js]: Yep, and I immediately felt bad for trying to refactor a working thing :)

**Pull request merged**